cloudferrylib/os/actions/prepare_networks.py

```diff
@@ -59,12 +64,22 @@ def run(self, info=None, **kwargs):
                     if src_net['floatingip']:
                         dst_flotingips = network_resource.get_floatingips()
                         dst_flotingips_map = \
-                            {fl_ip['floating_ip_address']: fl_ip['id'] for fl_ip in dst_flotingips}
-                        dst_floatingip_id = dst_flotingips_map[src_net['floatingip']]
-                        floating_ip = network_resource.update_floatingip(dst_floatingip_id, port['id'])
+                            {fl_ip['floating_ip_address']: \
+                                fl_ip['id'] for fl_ip in dst_flotingips}
+                        dst_floatingip_id = \
+                            dst_flotingips_map[src_net['floatingip']]
+                        floating_ip = \
```

Comment:
You shouldn't touch old code unless you introduce new or modify existing functionality. This way it's easier both for reviewers (less code to look through) and for you (less code to modify). In this particular case: 
1. Undo all the non-functional changes you made (whitespace, indentation, line breaks, etc)
2. Leave only functional changes and verify they don't break pep8